### PR TITLE
Activates GPU CI System

### PR DESCRIPTION
**Pull request by albert17**
Activates Blossom CI

### CODE REVIEW

File: .github/workflows/blossom-ci.yml [hunk 1]
```diff
@@ -67,6 +67,19 @@ jobs:
...
+      #  uses: actions/setup-java@v1
+      #  with:
+      #    java-version: 1.8
+
+      # add blackduck properties https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/631308372/Methods+for+Configuring+Analysis#Using-a-configuration-file
+      #- name: Setup blackduck properties
+      #  run: |
+      #       PROJECTS=$(mvn -am dependency:tree | grep maven-dependency-plugin | awk '{ out="com.nvidia:"$(NF-1);print out }' | grep rapids | xargs | sed -e 's/ /,/g')
+      #       echo detect.maven.build.command="-pl=$PROJECTS -am" >> application.properties
+      #       echo detect.maven.included.scopes=compile >> application.properties
```
Comment (benfred) [inline .github/workflows/blossom-ci.yml:81]: Can we just remove this block ? 
```suggestion
```

**Review by benfred — APPROVED** [review]
one minor comment - but looks good otherwise!

File: .github/workflows/blossom-ci.yml [hunk 2]
```diff
@@ -67,6 +67,7 @@ jobs:
           ref: ${{ fromJson(needs.Authorization.outputs.args).ref }}
           lfs: "true"
 
+
```
Comment (benfred) [inline .github/workflows/blossom-ci.yml:70]:
```suggestion
```

**Comment by albert17** [comment]
run blossom

**Pull request merged**